docker/mysql-to-postgres/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM ruby:2.6-stretch
```

Comment:
Could you make that whole thing a script, and call it from the Dockerfile(s) to avoid duplication?

Reply:
Done.